lib/core/utils/widgets/custom_toast.dart

```diff
@@ -7,7 +7,7 @@ class CustomToast {
     required String message,
     required Color backgroundColor,
     required ToastGravity gravity,
-    required IconData icon,
+    required dynamic icon,
```

Comment:
Refactor this from dynamic to Widget

to can use any widget like image asset or lottie file asset as json fuile

Reply:
An error appears as a result of this modification ( when I replace it with Widget )

Reply:
you can replace the icon with widget not Icondata